src/components/BookState.js

```diff
@@ -1,17 +1,34 @@
 import PropTypes from 'prop-types';
 import RemoveBookBtn from './DeleteBookBtn';
+import './BookState.css';
 
 export default function BookState({ book }) {
   return (
-    <div key={book.item_id}>
-      <p>Catagory</p>
-      <h3>{book.title}</h3>
-      <p>{book.author}</p>
-      <ul>
-        <li><button type="button">Comments</button></li>
-        <li><RemoveBookBtn id={book.item_id} /></li>
-        <li><button type="button">Edit</button></li>
-      </ul>
+    <div key={book.item_id} className="book-card">
+      <div className="book-details">
+        <p className="book-catagory">Catagory</p>
+        <h3 className="secondary-header">{book.title}</h3>
+        <p className="book-author">{book.author}</p>
+        <ul>
+          <li><button type="button">Comments</button></li>
+          <li><RemoveBookBtn id={book.item_id} /></li>
+          <li><button type="button">Edit</button></li>
+        </ul>
+      </div>
+      <div className="book-stutas">
+        <div className="circle-wrapper">
+          <div className="circle-progress" />
+        </div>
+        <div className="percentage">
+          <p>50%</p>
+          <p>completed</p>
+        </div>
```

Comment:
- [ ] Please add the right input for the percentage to match the design. It will be a placeholder but it should match to the actual design. Also, your text 'completed' is very low on contrast ratio. That makes it harder to read. Please apply the same font-color as the design.

yours|actual
---|---
![image](https://github.com/mahdinoori2000/bookstore/assets/91486791/27279431-103d-409b-8b22-2e0a21ba6aad)|![image](https://github.com/mahdinoori2000/bookstore/assets/91486791/9c56445b-6c72-43fd-9e39-ccfe8cc25735)